cssom-1/Overview.bs

```diff
@@ -375,7 +375,7 @@ To
  single SPACE (U+0020), followed by "<code>and</code>", followed by a single SPACE
  (U+0020), to <var>s</var>.
 
- <li>Sort the [=media features=] in lexicographical order.
+ Issue: The `and` bit needs updating for newer spec features.
```

Comment:
No, that's still correct; a type is *always* joined to the rest of the MQ with an `and`. You can't say `print or (grid)`.